### DISCUSSION TITLE
VueForm Upload Example Api 404

### DISCUSSION THREAD

**Discussion by FeiPengDev** [vueform/builder#80] [Questions] [answered]
🤔Maybe this is not a problem, I just found it when I was familiar with the file upload function.

![image](https://github.com/user-attachments/assets/17632880-627a-49e9-90b5-65f368637ae3)

**Comment by adamberecz** [comment, marked as answer]
Thanks for reporting but that's kind of expected as that's the default "dummy" upload endpoint. You can override them globally in `vueform.config.js`: https://vueform.com/docs/configuration#endpoints
